code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -2342,11 +2417,41 @@
 					if(spawning_horror == 1)
 						to_chat(H, "<b><span class='warning'>Something bursts out from inside you!</span></b>")
 						message_admins("[key_name(H)] has gibbed and spawned a new cyber horror due to nanobots. ([formatJumpTo(H)])")
-						H.visible_message("<b><span class='warning'>[H]'s body rips aparts to reveal something underneath!</b></span>")
-						new /mob/living/simple_animal/hostile/monster/cyber_horror(H.loc)
-						spawning_horror = 0
-						H.gib()
-
+						var/typepath = text2path(/mob/living/simple_animal/hostile/monster/cyber_horror/[H.species.name])
+							if(ispath(typepath))
```

Comment:
Whoops I left out the quotes, the typepath in the text2path section should have quotes